hicommonwealth/commonwealth
# Add contest log query

## Link to Issue
Closes: #8102

## Description of Changes
- Adds contest log query– which returns a log of onchain events (projections) for a given contest, merged with relevant data about the action/entity

## Test Plan
- Find a contest with some onchain activity, or create a new one
- Go to http://localhost:8080/api/v1/docs/#/Query/query.contest.getContestLog
  - Paste the contest address, run the query
  - Confirm that it returns an array of onchain events from the contest

## Deployment Plan
N/A

## Other Considerations
N/A
